ibis/__init__.py

```diff
@@ -33,6 +33,13 @@
     validator=ibis.config.is_bool,
 )
 ibis.config.register_option('default_backend', None)
+ibis.config.register_option(
+    'time_col',
```

Comment:
Can be a follow up. We should properly namespace this to be 
`context_adjustment.time_col`